conf/app.routes

```diff
@@ -661,6 +661,9 @@ POST        /:taxYear/rentals-rent-a-room/allowances/replacement-of-domestic-goo
 GET         /:taxYear/rentals-rent-a-room/allowances/change-replacement-of-domestic-goods               controllers.allowances.ReplacementOfDomesticGoodsController.onPageLoad(taxYear: Int, mode: Mode = CheckMode,  propertyType: PropertyType = RentalsRentARoom)
 POST        /:taxYear/rentals-rent-a-room/allowances/change-replacement-of-domestic-goods               controllers.allowances.ReplacementOfDomesticGoodsController.onSubmit(taxYear: Int, mode: Mode = CheckMode,  propertyType: PropertyType = RentalsRentARoom)
 
+GET         /:taxYear/rentals-rent-a-room/allowances/check-your-answers                                  controllers.rentalsandrentaroom.allowances.RentalsAndRentARoomAllowancesCheckYourAnswersController.onPageLoad(taxYear: Int)
```

Comment:
Allowances CYA page part of this ticket?

Reply:
No just putting the route in the correct place because it was in the SBA section